Formula/mallet.rb

```diff
@@ -0,0 +1,21 @@
+class Mallet < Formula
+  desc "MAchine Learning for LanguagE Toolkit"
+  homepage "http://mallet.cs.umass.edu/"
+  url "http://mallet.cs.umass.edu/dist/mallet-2.0.8.tar.gz"
+  sha256 "5b2d6fb9bcf600b1836b09881821a6781dd45a7d3032e61d7500d027a5b34faf"
+
+  bottle :unneeded
+
+  depends_on :java
+
+  def install
+    rm Dir["bin/*.{bat,dll,exe}"] # Remove all windows files
+    libexec.install Dir["*"]
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env)
+  end
+
+  test do
+    system "#{bin}/mallet | grep Mallet"
```

Comment:
I'm not sure what running `mallet` without any arguments exactly does. If that is something like just printing a usage or a version message, then we'd like something that exercises a deeper level of the functionality.

Reply:
Beside this, everything looks fine to me.

Reply:
Yes, there is a whole suite of commands that expect input. Would checking for help text from each of the commands be going deep enough? Fully testing functionality would require some test data. As a formula neophyte, I'm not sure how (or whether) that can be done.

Reply:
By going deep, we mean testing some real functionalities of the formula. According to the description `MAchine Learning for LanguagE Toolkit`, printing help messages is apparently not the real functionality the formula provides. 😁

If downloading additional fixtures is required, you may want to take a look at how `harfbuzz` handles this. https://github.com/Homebrew/homebrew-core/blob/master/Formula/harfbuzz.rb#L31-L34 and https://github.com/Homebrew/homebrew-core/blob/master/Formula/harfbuzz.rb#L57

Reply:
OK, I believe I found a decent one. This just covers one of the many functions, but a full test suite would be pretty large...